go/vt/vtgate/planbuilder/testdata/mirror_cases.json

```diff
@@ -39,6 +39,87 @@
       ]
     }
   },
+  {
+    "comment": "select unsharded, qualified, table mirrored to unsharded table with dual table",
+    "query": "select t1.id, 'hello' from unsharded_src1.t1 where t1.id = 1",
+    "plan": {
```

Comment:
this looks like reference test. Are we testing dual here?

Reply:
hm..not sure what i was thinking. i thought `dual` was showing up in the plan, but clearly it's not. will remove.

Reply:
fixed